confdgnmi/src/confd_gnmi_demo_adapter.py

```diff
@@ -11,9 +13,13 @@
 from confd_gnmi_common import make_xpath_path, make_gnmi_path
 
 log = logging.getLogger('confd_gnmi_demo_adapter')
+log.setLevel(logging.DEBUG)
```

Comment:
Should we keep DEBUG level?

Reply:
Removed, but we should revisit the logging levels, the current mechanism does not work correctly. (This line was added by you, btw.)